packages/blade/src/components/Radio/RadioIcon.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { useTheme } from '~components/BladeProvider';
 import Svg from '~components/Icons/_Svg';
 import { getIn } from '~utils';
-import { CheckboxIconWrapper } from '~components/Checkbox/CheckboxIcon/CheckboxIconWrapper';
+import { CheckboxIconWrapper as RadioIconWrapper } from '~components/Checkbox/CheckboxIcon/CheckboxIconWrapper';
```

Comment:
Lets move the `CheckboxIconWrapper` to a common place and use it for both Radio & Checkbox?

Reply:
Yep done in #631